tutorials/human_feedback/chatbot_with_human_feedback.ipynb

```diff
@@ -0,0 +1,263 @@
+    "        <img alt=\"phoenix logo\" src=\"https://raw.githubusercontent.com/Arize-ai/phoenix-assets/9e6101d95936f4bd4d390efc9ce646dc6937fb2d/images/socal/github-large-banner-phoenix.jpg\" width=\"1000\"/>\n",
+    "        <br>\n",
+    "        <br>\n",
+    "        <a href=\"https://docs.arize.com/phoenix/\">Docs</a>\n",
+    "        |\n",
+    "        <a href=\"https://github.com/Arize-ai/phoenix\">GitHub</a>\n",
+    "        |\n",
+    "        <a href=\"https://join.slack.com/t/arize-ai/shared_invite/zt-1px8dcmlf-fmThhDFD_V_48oU7ALan4Q\">Community</a>\n",
+    "    </p>\n",
+    "</center>\n",
+    "<h1 align=\"center\">Instrumenting a chatbot with human feedback</h1>\n",
+    "\n",
+    "Phoenix provides endpoints to associate user-provided feedback directly with OpenInference spans as annotations.\n",
+    "\n",
+    "In this tutorial, we will create a manually-instrument chatbot with user-triggered \"👍\" and \"👎\" feedback buttons. We will have those buttons trigger a callback that sends the user feedback to Phoenix and is viewable alongside the span. Automating associating feedback with spans is a powerful way to quickly focus on traces of your application that are not behaving as expected."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install \"phoenix\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "import os\n",
+    "import warnings\n",
+    "from typing import Any, Dict\n",
+    "from uuid import uuid4\n",
+    "\n",
+    "import httpx\n",
+    "import ipywidgets as widgets\n",
+    "import phoenix as px\n",
+    "from IPython.display import display\n",
+    "from openinference.semconv.trace import (\n",
+    "    OpenInferenceMimeTypeValues,\n",
+    "    OpenInferenceSpanKindValues,\n",
+    "    SpanAttributes,\n",
+    ")\n",
+    "from opentelemetry import trace as trace_api\n",
+    "from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter\n",
+    "from opentelemetry.sdk import trace as trace_sdk\n",
+    "from opentelemetry.sdk.trace.export import SimpleSpanProcessor"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from getpass import getpass\n",
+    "\n",
+    "if not (openai_api_key := os.getenv(\"OPENAI_API_KEY\")):\n",
+    "    openai_api_key = getpass(\"🔑 Enter your OpenAI API key: \")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define endpoints and configure OpenTelemetry tracing"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ENDPOINT = \"http://localhost:6006/v1\"\n",
+    "FEEDBACK_ENDPOINT = f\"{ENDPOINT}/span_annotations\"\n",
```

Comment:
sync=false?

Reply:
everything is fully sync in the example so I don't think we need it here—I was also planning to switch the default so we might not need to expose that explicitly